### ISSUE TITLE
Contents in main view panel flicker/vanish on button mouseover

### ISSUE THREAD

**Issue by carbeck** [dbeaver/dbeaver#4991]
#### System information: 
- Ubuntu 18.04.1 LTS
- DBeaver version 5.3.2.201901062110
- openjdk version "10.0.2" 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4, mixed mode)
- KDE plasmashell 5.12.7 with OpenGL 2.0 set as compositor (same problem with XRender, though)

#### Connection specification:
- psql (PostgreSQL) 10.6 (Ubuntu 10.6-0ubuntu0.18.04.1)
- Driver name: PostgreSQL
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No, the database runs locally (i.e. from localhost)

#### Describe the problem you're observing:
The main view panel which displays properties of the database, data tables, the SQL editor, etc. goes completely blank when hovering the mouse over a clickable control (buttons, tabs) within the panel. It comes back into view when hovering over a button again. Effectively, the view panel is thus flickering horribly as you move your mouse across the screen. This is very annoying and has persisted for the past 2 or 3 sub-versions.

#### Steps to reproduce, if exist:
* Open DBeaver
* Double-click on database
* Move mouse across screen
* Hover mouse over a button in the main view panel

#### Include any warning/errors/backtraces from the logs
https://gist.github.com/carbeck/ed97949cde2e693c0a34eee1c5fb96ca

**Comment by carbeck** [comment]
Yes, that makes it work. `GTK_IM_MODULE` is normally set to `xim` otherwise.

**Comment by lavalade** [comment]
I have had the same bug.  
To make BÉPO keyboard layout works properly, I switched from `ibus` to `xim`, but this have made DBeaver tabs blinking (especially SQL scripts).  
To fix this, I needed to do two things (need to restart the computer afterward):
1. `export GTK_IM_MODULE=ibus` as mentionned above, but also
2. `im-config` to rollback from `xim` to `ibus`

![Screenshot from 2023-05-30 10-06-50](https://github.com/dbeaver/dbeaver/assets/56821524/f184fefd-e4db-4ed2-9ed8-e4a96d7f781c)

My configuration:
- DBeaver Version 23.0.5.202305211610 (but same blinking bug with `xim` on previous version 22.3.3)
- Ubuntu 22.04.2 LTS
- GNOME 42.5
- X11
